README.md

```diff
@@ -46,7 +46,7 @@ after you cloned this repository to get the latest Countly iOS SDK.
 
 	* Change Bundle ID CodeSigning configurations according to your developer account configurations.
 	* Change Countly server URL and AppID: `config.appKey = @"YOUR_APP_KEY"; config.host = @"https://YOUR_COUNTLY_SERVER";`
-	* Enable push notifications: `config.features = @[CLYPushNotifications];`
+	* Enable push notifications: `config.features = @[CLYFeature.pushNotifications];`
```

Comment:
If this readme is common for all then we need to mention explicitly for Swift and Objective C.